test/conftest.py

```diff
@@ -0,0 +1,150 @@
+#  Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License").
+#  You may not use this file except in compliance with the License.
+#  A copy of the License is located at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  or in the "license" file accompanying this file. This file is distributed
+#  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+#  express or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+from __future__ import absolute_import
+
+import logging
+import os
+import platform
+import shutil
+import tempfile
+
+import boto3
+import pytest
+from sagemaker import LocalSession, Session
+
+logger = logging.getLogger(__name__)
+logging.getLogger('boto').setLevel(logging.INFO)
+logging.getLogger('botocore').setLevel(logging.INFO)
+logging.getLogger('factory.py').setLevel(logging.INFO)
+logging.getLogger('auth.py').setLevel(logging.INFO)
+logging.getLogger('connectionpool.py').setLevel(logging.INFO)
+
+SCRIPT_PATH = os.path.dirname(os.path.realpath(__file__))
+LATEST_COACH_VERSION = '0.11.0'
+LATEST_RAY_VERSION = '0.5.3'
```

Comment:
can these be read from somewhere instead?

Reply:
not really